app/brave_settings_strings.grdp

```diff
@@ -807,6 +807,16 @@
   <message name="IDS_SETTINGS_SITE_SETTINGS_SOLANA_BLOCK" desc="Label for solana site settings.">
     Block sites from accessing the Solana provider API
   </message>
+  <!-- Cardano -->
+  <message name="IDS_SETTINGS_SITE_SETTINGS_CARDANO" desc="Label for cardano site settings.">
+    Cardano
+  </message>
+  <message name="IDS_SETTINGS_SITE_SETTINGS_CARDANO_ASK" desc="Label for cardano site settings.">
+    Ask when a site wants to access the Cardano provider API
+  </message>
+  <message name="IDS_SETTINGS_SITE_SETTINGS_CARDANO_BLOCK" desc="Label for cardano site settings.">
+    Block sites from accessing the Cardano provider API
+  </message>
```

Comment:
nit. Do you think we could improve this `desc=` field to have a more meaningful description to the eventual person translating this? It looks a tad generic.

Reply:
Yes, please, at least let's say something like `Label for Cardano site settings radio button option to ask the user when sites wants to access Cardano API`, and `Label for Cardano site settings radio button option to block sites from accessing Cardano API`

Reply:
These are just copy-pastes from similar 
`IDS_SETTINGS_SITE_SETTINGS_ETHEREUM`
`IDS_SETTINGS_SITE_SETTINGS_SOLANA`
Show we also fix their descriptions?

Reply:
If you don't mind, that would be good. Helps with speeding up l10n.